src/mount/fuse/main.cc

```diff
@@ -427,6 +427,59 @@ static size_t count_colons_in_str(const char *str, size_t len) {
 	return colons_count;
 }
 
+/**
+ * @brief Normalize a single argument casing by converting uppercase characters
+ * in the options names to its lowercase equivalent.
+ */
+void normalize_argument_casing(char *arg) {
+	// A single argument looks like this: arg1=val1,arg2=val2,...,argn=valn
+	// So, case change should be applied at the begining and between a comma
+	// appearance and the next equal sign
+
+	bool should_apply_case_change = true;
+	char *current_char = arg;
+
+	auto to_lowercase_if_uppercase = [](char &c) {
+		if ('A' <= c && c <= 'Z') {
+			c = std::tolower(c);
+		}
+	};
+
+	while (*current_char) {
+		if (*current_char == ',') {
+			should_apply_case_change = true;
+		} else if (*current_char == '=') {
+			should_apply_case_change = false;
+		} else if (should_apply_case_change) {
+			to_lowercase_if_uppercase(*current_char);
+		}
+		current_char++;
+	}
+}
+
+/**
+ * @brief Normalize options casing by converting uppercase characters in the
+ * options names to its lowercase equivalent.
+ */
+void normalize_options_casing(fuse_args &args) {
```

Comment:
This function is a good candidate for unit testing. Please consider it.

Reply:
Done, I added a unittest. Please, check it in case some other improve could be necessary.